cider/exec/module/CiderRuntimeModule.cpp

```diff
@@ -98,6 +98,26 @@ CiderRuntimeModule::CiderRuntimeModule(
   }
 }
 
+std::unique_ptr<CiderBatch> CiderRuntimeModule::prepareOneBatchOutput(int64_t len) {
+  ArrowSchema* schema = CiderBatchUtils::convertCiderTableSchemaToArrowSchema(
+      ciderCompilationResult_->getOutputCiderTableSchema());
```

Comment:
Eventually `CiderTableSchema` should be replaced by `ArrowSchema` right?

Reply:
It's a option. Maybe could add more utils to make ```ArrowSchema``` build easily.